tests/unit/integrations/exclude-oembed-cache-post-type-test.php

```diff
@@ -43,7 +44,7 @@ public function set_up() {
 	public function test_register_hooks() {
 		Monkey\Filters\expectAdded( 'wpseo_indexable_excluded_post_types' )
 			->once()
-			->with( [ $this->instance, 'exclude_oembed_cache_post_type' ] );
+			->with( [ $this->instance, 'exclude_post_types' ] );
```

Comment:
I'm not sure if it makes sense to keep this test here. It doesn't cover a function in the 'DefaultClass' `\Yoast\WP\SEO\Integrations\Exclude_Oembed_Cache_Post_Type` and this test is duplicated in `tests/unit/integrations/abstract-exclude-post-type-test.php`.

(This would also apply to the other two classes)

Reply:
You're definitely right, I agree